x-pack/plugins/rule_registry/server/plugin.ts

```diff
@@ -133,7 +136,11 @@ export class RuleRegistryPlugin
 
     defineRoutes(router);
 
-    return { ruleDataService: this.ruleDataService };
+    return {
+      ruleDataService: this.ruleDataService,
+      createLifecycleRuleTypeFactory,
```

Comment:
Importing entities from a plugin in test is not well supported and breaks the CI.
Only way to use them in tests is exporting as public method/data etc.